app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,60 @@
+class TasksController < ApplicationController
```

Comment:
Nice! I like that you decided to adjust the name via the conventions accordingly